public/docs/_examples/dependency-injection/dart/lib/car/car_no_di.dart

```diff
@@ -13,10 +13,10 @@ class Car {
     engine = new Engine();
     tires = new Tires();
   }
-
   //#enddocregion car-ctor
 
   // Method using the engine and tires
-  drive() => '$description car with ${engine.cylinders} cylinders and ${tires.make} tires.';
+  drive() => '$description car with '
```

Comment:
Return type.

Also, this is against the general Dart style guide as we're using an action verb but not actually performing any changes.. Can we consider renaming this and making it a `get`?